### PR TITLE
Remove nullable disable in the osu.Desktop

### PR DESCRIPTION
**Pull request by andy840119**
Remove the nullable disable annotation in the `osu.Desktop` project.

### CODE REVIEW

File: osu.Desktop/LegacyIpc/LegacyIpcMessage.cs [hunk 1]
```diff
@@ -48,8 +46,8 @@ public new object Value
 
         public class Data
         {
-            public string MessageType { get; set; }
-            public object MessageData { get; set; }
+            public string MessageType { get; set; } = string.Empty;
+            public object MessageData { get; set; } = default!;
```
Comment (andy840119) [inline osu.Desktop/LegacyIpc/LegacyIpcMessage.cs:50]: Not really sure should use `new object()` instead?
The final goal is like https://github.com/dotnet/csharplang/issues/2328 and https://github.com/dotnet/csharplang/issues/3630 to make sure the setter must be assigned the value.
But seems there's no matched syntax for that 😢

Comment (peppy) [reply osu.Desktop/LegacyIpc/LegacyIpcMessage.cs:50]: Can you not make a `ctor` and use that instead?

Comment (andy840119) [reply osu.Desktop/LegacyIpc/LegacyIpcMessage.cs:50]: OK, i guess i will use that if there's no better choice 🙆🏼‍♂️

Comment (andy840119) [reply osu.Desktop/LegacyIpc/LegacyIpcMessage.cs]: Fixed.

**Review by frenzibyte — APPROVED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**